docs/manual/common/concepts/ComponentTechnologies.md

```diff
@@ -8,11 +8,11 @@ As a complete microservices platform, Lagom assembles a collection of technologi
 
     * As described in [[Implementing services|ServiceImplementation]], A Lagom service may be "simple" or "streamed".  Streaming, asynchronous Lagom services are built on top of [Akka Streams](https://doc.akka.io/docs/akka/2.5/stream/index.html).
 
-* Lightbend customers can additionally use [Enterprise Suite](https://www.lightbend.com/platform/production) components to operationalize and production-harden their systems.
+* [Lightbend Platform](https://www.lightbend.com/lightbend-platform) subscribers can use additional components to operationalize and production-harden their systems.
 
-    * Akka's [Split Brain Resolver](https://developer.lightbend.com/docs/akka-commercial-addons/current/split-brain-resolver.html) handles network failures and system crashes.
+    * Akka [Split Brain Resolver](https://developer.lightbend.com/docs/akka-commercial-addons/current/split-brain-resolver.html) handles network failures and system crashes.
```

Comment:
From Jamie's doc, this would become Akka Resilience enhancements? Then, you might add the word "include" because SBR is just part of them?

Reply:
Yes, I wasn't sure if the rest of those changes were definite yet, but after further discussion it sounds like they are, so I'll make some more updates.

Reply:
On second thought, I think I'd like to wait until we can roll that change out in other places. Right now if you were to search for "Akka Resilience Enhancements" you wouldn't find anything relevant.

Reply:
Yes, and to be clear "Akka Resilience Enhancements" is intended to be an umbrella for all of SBR, Kubernetes Lease, TSD, Config Checker and Diagnostics Recorder (all of which will keep their names).  (but not including the commercial persistence offerings).

For this section of the document - while it'd be ok to leave the Akka SBR bullet, it'd be nice to refer to Akka Resilience Enhancements which would point to htat suite of libraries (rather than just SBR), as they're all applicable to Lagom as well (I think).

I think it's ok to wait until the changes have been rolled out on the Commercial site, though this should be updated shortly after.

Reply:
Yes, I agree. I'll send a separate pull request afterwards.